diff --git a/rsty_physics/Cargo.toml b/rsty_physics/Cargo.toml
index 0054301..37d5ca6 100644
--- a/rsty_physics/Cargo.toml
+++ b/rsty_physics/Cargo.toml
@@ -10,4 +10,6 @@ crate-type = ["cdylib"]
 
 [dependencies]
 godot = { git = "https://github.com/thebigG/gdextension.git", branch = "2d_Line_beta6"}
-#godot = { path = "../../../godot", features = ["minimal"] }
+# godot = { path = "../../../godot", default-features = false, features = ["formatted", "convenience"] }
+
+#godot = { path = "/home/lgomez/gdextension/godot", features = ["minimal"] }
