diff --git a/rsty_physics/Cargo.toml b/rsty_physics/Cargo.toml
index 6ba7061..166e17c 100644
--- a/rsty_physics/Cargo.toml
+++ b/rsty_physics/Cargo.toml
@@ -11,10 +11,10 @@ crate-type = ["cdylib"]
 
 
 [dependencies]
-# godot = { git = "https://github.com/thebigG/gdextension.git", branch = "138_ref_count_fixes"}
+ godot = { git = "https://github.com/thebigG/gdextension.git", branch = "v0.4.2_wasm_fix", features = ["experimental-wasm", "lazy-function-tables"]}
 # godot = { git = "https://github.com/godot-rust/gdextension", branch = "master"}
 # godot = { git = "https://github.com/godot-rust/gdextension", rev ="d2de819", features = ["experimental-wasm", "lazy-function-tables"]}
-godot = {version= "0.4.1", features = ["experimental-wasm", "lazy-function-tables"]}
+#godot = {version= "0.4.2", features = ["experimental-wasm", "lazy-function-tables"]}
 rand = "0.8"
 #godot = { path = "../../../godot", features = ["minimal"] }
 #godot = { path = "../../projects/gdextension_array_debugging/gdextension/godot" }
